src/Hive.Core/sia380/sia380.py

```diff
@@ -232,6 +232,9 @@ def main(room_properties, room_schedules, floor_area, T_e_hourly, T_i_ub_hourly,
     Phi_P_per_m2 = room_properties["Waermeeintragsleistung Personen (bei 24.0 deg C, bzw. 70 W)"]
     Phi_L_per_m2 = room_properties["Waermeeintragsleistung der Raumbeleuchtung"]
     Phi_A_per_m2 = room_properties["Waermeeintragsleistung der Geraete"]
+    Q_dhw_year = room_properties["Jaehrlicher Waermebedarf fuer Warmwasser"] # in kWh/m2a
+    Q_dhw_hourly = (Q_dhw_year / HOURS_PER_YEAR) * floor_area # in kWh per hour
+    Q_dhw_monthly = [a*b for a, b in zip([Q_dhw_hourly * HOURS_PER_DAY] * MONTHS_PER_YEAR, DAYS_PER_MONTH)] # in kWh per month
```

Comment:
the `HOURS_PER_MONTH` on line 26 could be useful here 


```py
Q_dhw_monthly  = [Q_dhw_hourly * h for h in HOURS_PER_MONTH]
```

Reply:
right!